src/compiler/third-party-out-of-tree.md

```diff
@@ -0,0 +1,168 @@
+      compiler team members to agree. Prefer [licenses accepted by tidy][licenses], unless otherwise
+      required (ports of code from other projects, etc).
+- Abide by Rust's code of conduct.
+- Specify that the crate is maintained by the Rust compiler team and any appropriate working groups.
+    - In particular, this should detail the expected level of maintenance and stability for any
+      prospective users.
+    - This should also link to the working group details in this repository.
+- Be added to the list at the bottom of this page.
+- Follow semantic versioning.
+- Use `@bors` and `@rust-highfive`.
+- Use labels that are compatible with the existing triage process. This will allow nominated issues
+  in your out-of-tree crate to be discussed during triage meetings.
+    - eg. `T-compiler`, `I-nominated` (a full list is to be decided)
+
+### Is there a requirement for community infrastructure for an out-of-tree crate?
+There is no requirement that community infrastructure (such as Zulip servers/streams) be created for
+out-of-tree crates. This may be desirable if an out-of-tree crate gains a large community of
+contributors and users, but otherwise, the working group or compiler team streams should be used
+initially.
+
+Linkifiers for auto-linking to issues and PRs on the primary Rust Zulip server can be added on
+request.
+
+### Are there any recommendations for working with out-of-tree crates?
+Recommendations for working with out-of-tree crates will be documented in the rustc-dev-guide (see
+[rust-lang/rustc-dev-guide#285][guide_issue] for progress).
+
+### How should stabilization/semantic changes be handled in out-of-tree crates?
+It is important to involve the language team in any changes in out-of-tree crates that would result
+in stabilization or semantic changes to the language. Submodule changes in PRs to `rust-lang/rust`
+should be labelled appropriately (eg. `relnotes`, `t-compiler`, `t-lang`) just as if the change
+were implemented in `rust-lang/rust` directly, include a description of the changes when it is not
+obvious to those unfamiliar with the compiler or the out-of-tree crate.
+
+[licenses]: https://github.com/rust-lang/rust/blob/master/src/tools/tidy/src/deps.rs#L10-L19
+[guide_issue]: https://github.com/rust-lang/rustc-dev-guide/issues/285
+
+---
+
+In summary, the process for establishing an out-of-tree crate is as follows:
+
+1. Where appropriate, discuss and confirm the need within the working group for the out-of-tree
+   crate.
+1. Create a PR modifying this document to include the crate in the list below. Use
+   [`@rfcbot merge`](https://github.com/anp/rfcbot-rs#usage) to gain agreement from compiler
+   team members.
+1. Create a new repository in the `rust-lang` organization.
+    1. Navigate to `Settings > Collaborators & Teams` and add the `Compiler` team with write
+       privileges and `Assignees` with `read`.
+        - Individuals that are not part of the team can be added where appropriate.
```

Comment:
I think this should be done from the `team` repo and not manually.

Reply:
Fixed